.github/workflows/pr_runner.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
```

Comment:
Should these be 'main' instead of 'master' since it is running on the public repo?